clients/admin-ui/package.json

```diff
@@ -10,7 +10,7 @@
     "copy-export": "mkdir -p ../../src/fides/ui-build/static/admin/ && rsync -a --delete out/ ../../src/fides/ui-build/static/admin/",
     "cy:open": "cypress open",
     "cy:run": "cypress run",
-    "cy:start": "NODE_ENV=test next build && NODE_ENV=test next start",
+    "cy:start": "NODE_ENV=test next start",
```

Comment:
hmm I think we do still want the `build` here, otherwise next might not serve up the latest thing?

Reply:
Oh so I've moved this dependency on `build` to the root `turbo.json` file - https://github.com/ethyca/fides/pull/3105/files#diff-c8d74f2ac008193a3c9c6bfe765cccd0b4cf6413b857ff5ca4d8b27b6bee1ef0R41

Reply:
thanks for pointing that out! I guess this is something that wouldn't work well if you tried to use npm commands straight inside the admin-ui package (which I know you said not to do! just thinking about if I were a React dev looking at our code I'd probably hop right into admin-ui and try to run some of these commands). this one would be particularly tricky to figure out since you might still serve up a page, just not the latest version of the page.

perhaps you're already working on this, but a README in the `clients` folder really hammering in to use `turbo` could address this. also I know there's a `ui.md` doc that needs an all around update (not just for turbo, but it still references the fidesops repo, so it's very outdated!). I saw you replaced the npm commands with turbo commands, but we probably want a doc with a general overview of turbo and how we use it

Reply:
Good idea! I'll get that added